operator/roles/kiali-remove/tasks/main.yml

```diff
@@ -162,6 +162,7 @@
   with_items:
   - "{{ query('k8s', namespace=kiali_vars.deployment.namespace, kind='OAuthClient', resource_name='kiali-' + kiali_vars.deployment.namespace, api_version='oauth.openshift.io/v1') if is_openshift == True else [] }}"
   - "{{ query('k8s', namespace=kiali_vars.deployment.namespace, kind='Route', resource_name='kiali', api_version='route.openshift.io/v1') if is_openshift == True else [] }}"
+  - "{{ query('k8s', kind='ConsoleLink', label_selector='app=kiali')}}"
```

Comment:
I just realized this is wrong too - because in a multi-tenant scenario, if you remove one Kiali CR from one tenant, this is going to delete every consolelink for every tenant. Not good.

Reply:
we might need to add another label to our ConsoleLinks - whatever its name is, the value should be the deployment namespace of Kiali.. so we can add it to the label selector here. something like

`label_selector='app=kiali tenant='+kiali_vars.deployment.namespace`